cmd/pr.go

```diff
@@ -34,11 +34,12 @@ func createPR() {
 
 	issueSummary := jiraService.GetSummaryForIssueId(issueId)
 	prTitle := fmt.Sprintf("%v %v", issueId, issueSummary)
+	prBody := jiraService.FormatIssueURL(issueId)
 
 	services.GitService{}.PushCurrentBranch()
 
-        url := services.GitHubService{}.CreatePullRequest(prTitle)
-        fmt.Println(url)
+	url := services.GitHubService{}.CreatePullRequest(prTitle,prBody)
+	fmt.Println(url)
```

Comment:
This was indented with spaces, but the rest of the file used tabs, This seems to differ per file though. `JiraService.go` uses spaces all over.

Reply:
Yes I am using vim for go instead of idea because of financial issues :joy: So things might not be as consistent as I would like. Thx for making me aware of this problem.